lib/src/models/feedItem.dart

```diff
@@ -0,0 +1,34 @@
+import 'homeworkItem.dart';
+import 'meeting.dart';
+
+enum FeedItemType { Homework, Lecture, Meeting }
+
+extension FeedItemTypeExtension on FeedItemType {
+  static FeedItemType? fromItem(dynamic item) {
+    if (item is HomeworkItem) {
+      return FeedItemType.Homework;
+    } else if (item is Meeting) {
+      return FeedItemType.Meeting;
+    }
```

Comment:
```suggestion
    } else {
      return FeedItemType.Lecture
    }
```


Just a guess from me, haven't tested to see if it would work.

Reply:
I was looking to add this but I think I'll wait until a LectureItem is created, otherwise there won't be any way to receive such a type.

Reply:
It's better to put this in now even if it won't function than not and hope you remember later. I can't even start to count the number of times I've done the latter to my own detriment.